docs/DeveloperGuide.md

```diff
@@ -423,8 +426,70 @@ Use Case ends.
 2. CCAConnect displays all valid commands.
 
 Use case ends.
+<br>
+
+<br>
+**Use case: UC07 - View all members**
+**Actor: User**
+**MSS**
+
+1. User requests to see all members in the contacts.
+2. CCAConnect displays all members.
+
+Use case ends.
+<br>
+
+<br>
+**Use case: UC08 - Mark attendance of members**
+**Actor: User**
+**MSS**
+
+1. User requests to mark the attendance of members.
+2. User provides the details of the session (date) to mark attendance, and list of telegrams of members to be marked.
+3. System marks the attendance of the list of members on the specified date.
+
+Use case ends.
+
+**Extensions**
+* 8a. System detects that user did not enter all the required fields.
```

Comment:
I think the numbering should match the step numbers. so 1a in this case

Reply:
Thanks, already updated.